tests/Oryx.BuildImage.Tests/DotNetCore/DotNetCoreDynamicInstallTest.cs

```diff
@@ -590,13 +590,23 @@ public void BuildsApplication_ByDynamicallyInstallingSDKs_IntoCustomDynamicInsta
                     NetCore7PreviewMvcApp,
                     imageHelper.GetGitHubActionsBuildImage("github-actions-buster"));
 
-
-                //buullseye
+                //bullseye
                 data.Add(
                     DotNetCoreRunTimeVersions.NetCoreApp31,
                     DotNetCoreSdkVersions.DotNetCore31SdkVersion,
                     NetCoreApp31MvcApp,
                     imageHelper.GetGitHubActionsBuildImage("github-actions-bullseye"));
+                data.Add(
+                    DotNetCoreRunTimeVersions.NetCoreApp60,
+                    DotNetCoreSdkVersions.DotNet60SdkVersion,
+                    NetCore6PreviewWebApp,
+                    imageHelper.GetGitHubActionsBuildImage("github-actions-bullseye"));
+                data.Add(
+                    DotNetCoreRunTimeVersions.NetCoreApp70,
+                    DotNetCoreSdkVersions.DotNet70SdkVersion,
+                    NetCore7PreviewMvcApp,
+                    imageHelper.GetGitHubActionsBuildImage("github-actions-bullseye"));
+                
```

Comment:
Validation job should fail for these tests until the Platform binaries job is run to upload to bullseye SDKs to dev.